### PR TITLE
[docs] added feature guide for tf logs conversion & params auto tracking

### PR DESCRIPTION
**Pull request by devfox-se**
added guides to
- #1272 
- #1203

### CODE REVIEW

File: docs/source/guides/integrations/basic_aim_tensorflow_event_conversion.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+the conversion process.
+
+Please note that only the following TF plugins are currently supported 
+- scalar
+- image
+
+To convert TensorFlow events, `aim convert` commant must be run on your log directory.
+
+```commandline
+aim convert --repo .aim tf --logdir ~/tensorflow/logdir
```
Comment (alberttorosyan) [inline docs/source/guides/integrations/basic_aim_tensorflow_event_conversion.md:16]: better to leave out `.aim` (it is handled automatically)
`aim convert --repo aim_repo tf --logdir ~/tensorflow/logdir` or
`aim convert --repo . tf --logdir ~/tensorflow/logdir`

File: docs/source/guides/integrations/basic_aim_tensorflow_event_conversion.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
+### Show TensorFlow events in Aim 
+
+Aim give your possibility to convert TensorFlow event files into native format
```
Comment (alberttorosyan) [inline docs/source/guides/integrations/basic_aim_tensorflow_event_conversion.md:3]: Aim gives you...

**Review by alberttorosyan — COMMENTED** [review]
left a couple of comments
the rest looks good

File: docs/source/guides/integrations/basic_aim_tensorflow_event_conversion.md (hunk 1 above)
Comment (devfox-se) [reply docs/source/guides/integrations/basic_aim_tensorflow_event_conversion.md]: I removed `--repo` at all since it's handled automatically

**Review by alberttorosyan — APPROVED** [review]
lgtm

**Pull request merged**